main.js

```diff
@@ -0,0 +1,29 @@
+const generateNumbers = () => {
+    const generate3to5 = () => {
+        return Math.floor(Math.random() * 3) + 3
+    };
+    let numbersOnScreen = generate3to5();
+    let allNumbers = [];
+    for (i=0; i<numbersOnScreen; i++) {
+        let randNumber = Math.floor(Math.random() * 100.1);
```

Comment:
i thought about this randomly since finishing friday but it actually needs to multiply by 101 not 100.1. Will change it quickly monday morning.